benchmark_cutlass_linear_e2e.py

```diff
@@ -17,24 +17,24 @@ def forward(self, x):
 
 
 class CUTLASSSparseLinear(nn.Module):
-    def __init__(self, linear_model):
+    def __init__(self, linear_model, n):
         super().__init__()
 
         m, k = linear_model.weight.shape
         mask = linear_model.weight.data != 0
 
         self.weight_tensor = linear_model.weight.data.masked_select(mask).view(m, k // 2)
-        self.bias_tensor = linear_model.bias.data
+        self.bias_tensor = linear_model.bias.repeat((n, 1)).T.contiguous()
```

Comment:
Is it possible to avoid this repeat by using 0-valued strides within CUTLASS? As in, you can use [`expand`](https://pytorch.org/docs/stable/generated/torch.Tensor.expand.html#torch.Tensor.expand) instead of `repeat`.

Reply:
I may be wrong, but I don't see how to avoid repeating explicitly...  Namely, we want to calculate $x\cdot A^{T}+b$, where $x$ is input, $A$ is weight matrix and $b$ is bias vector.  We're calculating the product as $(A\cdot x^{T})^{T}$, so we'd have to calculate the whole thing as $(A\cdot x^{T}+b^{T})^{T}$.  Thus, $b$ would have to be treated as in column-major order, more precisely with row stride 1 and column stride 0.  However, apparently in CUTLASS the layout of this particular matrix (that, btw, has to be same as format of output matrix) could be row-major only for sparse GEMM.  (Note that in CUTLASS, layouts are "more explicit" than in PyTorch, for example "row-major" in CUTLASS means one could specify only the row stride, while column stride is implicitly assumed to be 1.)

Reply:
So we need a repeated b, let's call it B, but in transposed form B^T of shape (M, N). We have a `b` of shape (N,). We can have a B of shape (N, M) with striding (1, 0) via `.unsqueeze(-1).expand(N, M)`. Then B^T will have striding (0, 1) and shape (M, N). So if we can specify the column stride, if I'm understanding this correctly, we could set it to 0 after passing in `b.unsqueeze(-1).expand(N, M).t()`?

Reply:
In the context of linear layer, we have $K$ inputs and $M$ outputs, so $A$ is of shape $(M,K)$, while input $x$ is $(N,K)$, and $b$ is $(M,)$ - this is all taken from `nn.Linear`.  We need to calculate $x\cdot A^{T}+b$, that we do through $(A\cdot x^{T}+b^{T})^{T}$.  Thus,  $b^{T}$ when expanded has to be of shape $(M,N)$.  Here, `b.unsqueeze(-1).expand(M, N)` provides what is needed, however its strides are $(1,0)$ instead of $(0,1)$ that would be fit for what CUTLASS can do.

Reply:
We could also use a custom epilogue instead of trying to squeeze this into the out argument so to speak. I wonder if you can pass beta as a pointer to [LinearCombination](https://github.com/NVIDIA/cutlass/blob/af332d4aa9ee19f901f0ceac442078fb04b350fb/include/cutlass/epilogue/thread/linear_combination.h#L122).